releasenotes/notes/gunicorn-issue-09308901fa00e76c.yaml

```diff
@@ -0,0 +1,3 @@
+---
+prelude: >
+  gunicorn: known issue: ddtrace-run does not work with gunicorn
```

Comment:
Is this release note required? This issue has been documented for a few years now. Calling it out in a new release note might cause confusion

Reply:
🤷 I don't really know the norms, but including it in a release note seems reasonable to me. @brettlangdon suggested that I should add a release note.

Reply:
I think it makes sense to include as a known issue since the previous suggestion was that yes you could use `ddtrace-run` with gunciorn and gevent, but now you cannot use them together.